src/meta/src/controller/streaming_job.rs

```diff
@@ -198,10 +196,6 @@ impl CatalogController {
                 table.id = job_id as _;
                 let table_model: table::ActiveModel = table.clone().into();
                 Table::insert(table_model).exec(&txn).await?;
-
-                objects.push(PbObject {
-                    object_info: Some(PbObjectInfo::Table(table.to_owned())),
-                });
```

Comment:
It seems in the old code, we notify the MV table + internal tables, but now we just notify internal tables. Why don't we notify the MV table any more? 🤔 Did I miss sth?

Reply:
See: https://github.com/risingwavelabs/risingwave/pull/20460/files#r1959043203